### ISSUE TITLE
Lineage Graph Ignores Sources that are Tagged

### ISSUE THREAD

**Issue by alittlesliceoftom** [fishtown-analytics/dbt#3124]
### Describe the bug
GIVEN THAT my yaml attempts to tag a set of sources like: 

```
 - name: raw_location
   tags: 
     - DailySync
```

Then when I build and serve docs, the sources exists, and have the tag, but they don't show on the lineage graph's. 

### Steps To Reproduce
Add a tag at the source name level to your yaml

```
 - name: raw_location
   tags: 
     - DailySync
```

### Expected behavior
See all sources. 

### Screenshots and log output
Not included as work data

### System information
**Which database are you using dbt with?**
- [ ] postgres
- [ ] redshift
- [ ] bigquery
- [ ] snowflake
- [x] other (specify: Synapse)

**The output of `dbt --version`:**
```
  - sqlserver: 0.18.1
```

**The operating system you're using:**
Windows (10) + Unix deployments. 
**The output of `python --version`:**
Python 3.8.7

**Comment by alittlesliceoftom** [comment]
Relationships between source and model hold, just not on graph.

**Comment by jtcohen6** [comment]
@alittlesliceoftom Good news! This bug was resolved for v0.19.0 (https://github.com/fishtown-analytics/dbt-docs/pull/139). Could you try upgrading and ensure it resolves the issue?
